modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,17 +19,80 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Replace hard-coded image references with environment variables in the following files:
 
-.. Update `spec.relatedImages`:
+** For Go- and Ansible-based Operator projects, update the `roles/memcached/tasks/main.yml` file:
```

Comment:
Go-based Operator doesn't have the file roles/memcached/tasks/main.yml. Can update the image in example file [controllers/memcached_controller.go](https://docs.openshift.com/container-platform/4.9/operators/operator_sdk/golang/osdk-golang-tutorial.html#osdk-golang-implement-controller_osdk-golang-tutorial)

Update the image to be the digest type.
Image: "memcached:1.4.36-alpine",
Image:   "docker.io/memcached@sha256:00b68b00139155817a8b1d69d74865563def06b3af1e6fc79ac541a1b2f6b961",